src/redux/reducers.js

```diff
@@ -0,0 +1,44 @@
+import { createReducer } from "reduxsauce";
+import { Types } from "./actions";
+import { getSimplifiedModuleInfo } from "../utils/getSimplifiedModuleInfo";
+
+export const INITIAL_STATE = {
+  currentPlan: "sample plan",
+  plans: { "sample plan": {}, "second major": {}, "without hons": {} },
+  modules: getSimplifiedModuleInfo(),
+};
+
+export const changePlan = (state = INITIAL_STATE, { name }) => {
+  return { ...state, currentPlan: name };
+};
+
+export const addPlan = (state = INITIAL_STATE, { plan }) => {
+  const plans = { ...state.plans, ...plan };
+  return { ...state, plans };
+};
+
+export const renamePlan = (state = INITIAL_STATE, { prevName, newName }) => {
```

Comment:
Is there a popup in place to rename?

Reply:
Not yet I think, but the action is there so we could use it in the future.